deploy/operator.yaml

```diff
@@ -60,6 +60,10 @@ spec:
                 configMapKeyRef:
                   name: nmstate-config
                   key: interfaces_filter
+            - name: ENABLE_PROFILER
+                value: "False"
+            - name: PROFILER_PORT
```

Comment:
Please add it to the port section of the pod: https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/